weave-scope/README.md

```diff
@@ -0,0 +1,185 @@
+~~~
+
+Check that all seven application containers are running by typing `docker ps`:
+
+~~~bash
+CONTAINER ID  IMAGE            PORTS                 NAMES
+fe41c10a63ca  tns_lb1:latest   0.0.0.0:8001->80/tcp  tns_lb1_1
+c94005d87115  tns_lb2:latest   0.0.0.0:8002->80/tcp  tns_lb2_1
+8c15a1325094  tns_app1:latest  8080/tcp              tns_app1_1
+645386356a2e  tns_app2:latest  8080/tcp              tns_app2_1
+e34ccea042fd  tns_db3:latest   9000/tcp              tns_db3_1
+c0d53d1327b4  tns_db2:latest   9000/tcp              tns_db2_1
+0a920e17818a  tns_db1:latest   9000/tcp              tns_db1_1
+~~~
+
+Verify that the containers are reachable by curling one of the tns_lb instances. 
+
+By default, the containers listen on ports 8001 and 8002:
+
+~~~bash
+$ curl localhost:8001  # on a Mac, try: curl `boot2docker ip`:8001
+lb-6d5b2352f76d4a807423ce847b80f060 via http://app1:8080
+app-60fbe0a31aee9526385d8e5b44d46afb via http://db2:9000
+db-e68d33ceeddbb77f4e36a447513367e8 OK
+~~~
+
+With just a few commands, you've deployed a three-tier application stack into seven containers. 
+
+###Launching Weave Scope
+
+With the sample app up and running, you are ready to install and launch Weave Scope:
+
+~~~bash
+sudo wget -O /usr/local/bin/scope \
+  https://github.com/weaveworks/scope/releases/download/latest_release/scope
+sudo chmod a+x /usr/local/bin/scope
+sudo scope launch
+~~~
+
+Point your web browser to: `http://192.168.99.100:4040/` (or to the address displayed to you in the terminal window after Weave Scope was launched).
+
+
+>Note: If you are running containers on multiple hosts, you must launch Weave Scope onto each host. But if you are running Weave Net to network containers, you can launch Weave Scope onto one host and Weave Net, provided you have weaveDNS running will do the rest, see Launching Weave Scope When You are Using Weave Net, below. 
+
+
+##Launching Weave Scope When Using Weave Net to Connect Containers
+
+If you are running Weave Scope on the same machine as the Weave network, the probe uses `weaveDNS` to automatically discover other apps and containers on your network. 
+
+Weave Scope registers itself using the address `scope.weave.local`. Each probe then, sends its report to every app registered under this address. In other words, if you are running Weave Net with DNS, you do not need to take any further steps. Launch once and Weave Net takes care of the rest. 
+
+If you don't want to use weaveDNS, you can instruct Weave Scope to cluster with other Weave Scope instances on the command line. Hostnames and IP addresses are acceptable, both with and without ports, for example:
+
+~~~bash
+scope launch scope1:4030 192.168.0.12 192.168.0.11:4030
+~~~
+
+Hostnames are regularly resolved as A records, where each answer is used as a target.
+
+##Visualizing Your Infrastructure
+
+Once Weave Scope is displayed in your browser, it gives you an immediate overview of your network, and shows all of your containerized apps. 
```

Comment:
You might want to mention that the `scope launch` command will print out the URLs at which the Scope UI is available.